### DISCUSSION TITLE
Hackathon: 2024-04-30

### DISCUSSION THREAD

**Discussion by stefaniebutland** [earthaccess-dev/earthaccess#548] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by mfisher87** [comment]
I forgot to mention the hack day thread today during wrap-up :grimacing:
